src/machine/machined-varlink.c

```diff
@@ -688,7 +689,8 @@ static int manager_varlink_init_machine(Manager *m) {
                         "io.systemd.Machine.Unregister",  vl_method_unregister,
                         "io.systemd.Machine.Terminate",   vl_method_terminate,
                         "io.systemd.Machine.Kill",        vl_method_kill,
-                        "io.systemd.MachineImage.List",   vl_method_list_images);
+                        "io.systemd.MachineImage.List",   vl_method_list_images,
+                        "io.systemd.MachineImage.Update", vl_method_update_image);
```

Comment:
hmm, could you also add `r = sd_varlink_server_listen_address(s, "/run/systemd/machine/io.systemd.MachineImage", 0666);` below, so that we have a different entrypoint for the 2nd interface (even if we server everything on both).

(And then update the test case to use the 2nd entrypoint, just to be fully correct)